libraries/chain/transaction_context.cpp

```diff
@@ -459,7 +459,7 @@ namespace eosio { namespace chain {
                const int64_t cpu_limit = (cpu_limited_by_account ? account_cpu_limit : objective_duration_limit.count());
                EOS_ASSERT( billed_us <= cpu_limit,
                            tx_cpu_usage_exceeded,
-                           "${desc} CPU time (${billed} us) is greater than the maximum billable CPU time for the transaction (${billable} us)",
+                           "${desc} Transaction CPU time (${billed} us) is greater than the maximum available (${billable} us). Please stake more resources to CPU.",
```

Comment:
This isn't alway accurate.  This message will also appear when the cpu time exceeds the chain's configured maximum transaction CPU usage.  Also, advice of how to fix the error doesn't really belong here, as it depends on the system contract.
- "Transaction" should not be capitalized.
- The other messages from this function should be consistent.

Reply:
@jafri:

`cpu_limited_by_account` can be used to determine whether the limit was reached because the account's quota is too low (and therefore it may help to recommend increasing the account's CPU quota) or if it is due to other objective reasons (e.g. the limit was determined by either the maximum transaction CPU consensus parameter or by the maximum specified in the transaction header).

And as @swatanabe-b1 mentioned, the language here should not be specific to any particular system contract. So even the word "stake" isn't  really appropriate. Personally, I think a phrase like "increasing CPU resources may help" would be acceptable, but only in the case where we know the limit was reached due to a billed account's quota (i.e. if `cpu_limited_by_account` is true).